stable/rocketchat/templates/svc.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  ports:
+  - name: http
+    port: 3000
+    targetPort: http
+  selector:
+    app: {{ template "fullname" . }}
```

Comment:
After changing the `app` label as mentioned above, the selector should be as follows:

```yaml
app: {{ template "name" . }}
release: "{{ .Release.Name }}"
```